### PR TITLE
Add textattack, wandb, bert_score, language-tool-python, flair, nlp, sentence-transformers

### PR DESCRIPTION
**Pull request by ndmaxar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ndmaxar** [comment]
@rluria14 and @oblute ok to be maintainers?

File: recipes/bert_score/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - bert-score-show=bert_score_cli.visualize:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib
```
Comment (scopatz) [inline recipes/bert_score/meta.yaml:25]: This should be `matplotlib-base`

File: recipes/language-tool-python/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/jxmorris12/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ee548908e37af66a7bce6ad53b47d4427def2eb6194be4faa36b63711bb56c13
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/language-tool-python/meta.yaml:14]: This can be noarch: python

File: recipes/sentence-transformers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  patches:
+    - BERT.py.patch
+    - RoBERTa.py.patch
+    - SentenceTransformer.py.patch
+    - XLNet.py.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
```
Comment (scopatz) [inline recipes/sentence-transformers/meta.yaml:20]: This can be noarch python

File: recipes/textattack/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/QData/{{ name|title }}/archive/{{ version }}.tar.gz
+  sha256: 9ccb8b2a43120ee2d140fda78d4dfd7f0339e6d051a2c67cc9e891df2b993d85
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux or py<37]
```
Comment (scopatz) [inline recipes/textattack/meta.yaml:15]: Same, noarch

File: recipes/wandb/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  sha256: 0ed543a5f2df3df6a2f53a52f7ab7103c43d014b7ec87b553df8c3cec0935206
+
+build:
+  number: 0
+  entry_points:
+    - wandb=wandb.cli:cli
+    - wb=wandb.cli:cli
+    - wanbd=wandb.cli:cli
+    - wandb-docker-run=wandb.cli:docker_run
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/wandb/meta.yaml:19]: same noarch

File: recipes/textattack/meta.yaml (hunk 4 above)
Comment (ndmaxar) [reply recipes/textattack/meta.yaml]: Should I keep the skip? I'm still fuzzy on how exactly noarch and skips work together.

File: recipes/bert_score/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 02114fb1a331131d4f4a70cbbc3022f4b873ef434b18da30cc0d7b7d5822834a
+
+build:
+  number: 0
+  entry_points:
+    - bert-score=bert_score_cli.score:main
+    - bert-score-show=bert_score_cli.visualize:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
```
Comment (scopatz) [inline recipes/bert_score/meta.yaml:18]: This should be noarch:python

Comment (scopatz) [reply recipes/bert_score/meta.yaml:18]: nm sorry

File: recipes/nlp/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dill
+    - filelock
+    - numpy
+    - pyarrow ==0.17.1
```
Comment (scopatz) [inline recipes/nlp/meta.yaml:25]: Does this pin need to be so tight?

Comment (ndmaxar) [reply recipes/nlp/meta.yaml:25]: Yeah. It has to be this or `==0.17`. Not sure why though. Only `>=0.16` is needed by nlp.

**Comment by scopatz** [comment]
Is this ready to go?

**Comment by ndmaxar** [comment]
Yeah I think so.

**Comment by scopatz** [comment]
OK - In the future, please put in PRs that share all of the same build platforms

**Pull request merged**